open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/pom.xml

```diff
@@ -75,6 +75,12 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
```

Comment:
The 'server' component MUST NOT have a dependency on spring. We need to be able to support other rest bindings in future (such as microprofile) and keep all spring capability in the spring specific module

Reply:
Agreed